.gitignore

```diff
@@ -1,12 +1,9 @@
 # Copyright (c) Microsoft Corporation.
 # SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
 
-# python
-__pycache__/
-*.py[cdo]
-
 .vs/
 .vscode/
 /build/
 /out/
+__pycache__/

```

Comment:
I think this list should simply be VSCode-sorted (so a non-`/`-prefixed directory wouldn't be mixed in) although it isn't critical.